src/lightning_lite/accelerators/registry.py

```diff
@@ -112,11 +112,8 @@ def __str__(self) -> str:
         return "Registered Accelerators: {}".format(", ".join(self.available_accelerators()))
 
 
-AcceleratorRegistry = _AcceleratorRegistry()
-
-
-def call_register_accelerators(base_module: str) -> None:
```

Comment:
This now takes the registry as input, because there are two now: One for Lite, one for PL.